src/client/components/Sidebar.tsx

```diff
@@ -39,7 +49,14 @@ class Sidebar extends PureComponent<SidebarProps, SidebarState> {
       deviceOS: '',
       deviceInfo: '',
       userMessage: '',
-      userName: ''
+      userName: '',
+      timezone: {
+        value: 'GMT',
+        label: '(GMT+0:00) UTC',
+        abbrev: 'GMT',
+        offset: 0,
+        altName: 'GMT'
```

Comment:
This can be a `const` object at the top of the file.

Reply:
As requested, a `const` object was created with the initial timezone values.